llvm/lib/Transforms/Scalar/DeadStoreElimination.cpp

```diff
@@ -2132,182 +2177,182 @@ struct DSEState {
     }
     return MadeChange;
   }
-};
 
-static bool eliminateDeadStores(Function &F, AliasAnalysis &AA, MemorySSA &MSSA,
-                                DominatorTree &DT, PostDominatorTree &PDT,
-                                const TargetLibraryInfo &TLI,
-                                const LoopInfo &LI) {
-  bool MadeChange = false;
+  // Try to eliminate dead defs killed by `KillingLocWrapper` and return the
```

Comment:
IIUC, `Shortened` really means "KillingI was deleted" (please correct me if I'm wrong). So maybe change the variable name to `DeletedKillingLoc` and the comment to

```
Try to eliminate dead defs killed by `KillingLocWrapper`. Return whether any changes were made, and whether `KillingLocWrapper` was deleted.
```

Reply:
Done!